xml/System.IO/TextWriter.xml

```diff
@@ -439,10 +439,20 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="writers">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="writers">The <see cref="T:System.IO.TextWriter" /> instances to which all operations should be broadcast (multiplexed).</param>
+        <summary>Creates an instance of <see cref="T:System.IO.TextWriter" /> that writes supplied inputs to each of the writers in <paramref name="writers" />.</summary>
+        <returns>An instance of <see cref="T:System.IO.TextWriter" /> that writes supplied inputs to each of the writers in <paramref name="writers" /></returns>
+        <remarks>
+          <para>The resulting instance will delegate each operation to each of the writers in <paramref name="writers" />.</para>
+          <para>For example, calling <see cref="M:System.IO.TextWriter.Write(System.Char)" /> will write the specified char to each writer, one after the</para>
+          <para>other. The writers will be written to in the same order as they are specified in <paramref name="writers" />.</para>
+          <para>An exception from the operation on one writer will prevent the operation from being performed on subsequent writers.</para>
+          <para>
```

Comment:
This doesn't make sense: 
```
one after the</para>
<para>other.
```
Can the porting tool be adjusted to only wrap sentences with full stop periods (`.\n`)?

Reply:
My guess is that the tool is just reading the newlines manually inserted and putting them in as individual paras. But really it should look for two consecutive newlines, like markdown does.

Reply:
^ What @adegeo said. The tool isn't joining sentences. I got feedback that the tool was not respecting newlines due to the absence of `<para>`.